test/test_plugin.py

```
@@ -34,810 +20,174 @@
 
     @qml.qjit(
         target="mlir",
-        pipelines=DEFAULT_PIPELINE,
-        autograph=True,
+        pipelines=DIRECT_PIPELINE,
         keep_intermediate=2,
         pass_plugins={MQT_PLUGIN_PATH},
         dialect_plugins={MQT_PLUGIN_PATH},
     )
-    def module() -> Any:  # ruff:ignore[any-type]
+    def module() -> Any:  # ruff: ignore[any-type]
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `module`

(ANN401)